prow/config.yaml

```diff
@@ -42,6 +43,92 @@ presubmits:
     context: pull-charts-e2e
     rerun_command: "/test pull-charts-e2e"
     trigger: "(?m)^/test( all| pull-charts-e2e),?(\\s+|$)"
+  kubernetes/federation:
```

Comment:
I recommend splitting this into two PRs: one to add this job and then another to enable tide.

Reply:
Done.

Reply:
tide PR is: https://github.com/kubernetes/test-infra/pull/5167